app/src/main/java/ch/hikemate/app/ui/theme/Color.kt

```diff
@@ -3,6 +3,7 @@ package ch.hikemate.app.ui.theme
 import androidx.compose.ui.graphics.Color
 
 val primaryColor = Color(0xFF3B9DE8)
+val plannedDateMessageColor = Color(0xFF3B82F6)
```

Comment:
Question: Why don't you use the already defined colors?

Reply:
Because it is not the same color, and my task did not involve changing the color itself, so I kept it as-is.

Reply:
Can you please use the primaryColor instead?

Reply:
Done 👌🏻